lib/roo/csv.rb

```diff
@@ -90,17 +90,36 @@ def read_cells(sheet = default_sheet)
     def each_row(options, &block)
       if uri?(filename)
         each_row_using_tempdir(options, &block)
-      elsif is_stream?(filename_or_stream)
-        ::CSV.new(filename_or_stream, options).each(&block)
       else
-        ::CSV.foreach(filename, options, &block)
+        csv_foreach(filename_or_stream, options, &block)
       end
     end
 
     def each_row_using_tempdir(options, &block)
       ::Dir.mktmpdir(Roo::TEMP_PREFIX, ENV["ROO_TMP"]) do |tmpdir|
         tmp_filename = download_uri(filename, tmpdir)
-        ::CSV.foreach(tmp_filename, options, &block)
+        csv_foreach(tmp_filename, options, &block)
+      end
+    end
+
+    # From Ruby 2.5, options argument of CSV.new/CSV.foreach is a keyword argument.
+    # Before Ruby 2.5, that argument is a Hash.
+    # Therefore, this workaround can be removed if Ruby 2.3 and 2.4 are dropped.
```

Comment:
Yes, please remove such workaround.

Reply:
Can I edit `gemspec` and `.travis.yml` to drop ruby 2.3 & 2.4 in this branch: `ruby_27_support`?

Reply:
I removed WA for Ruby 2.3 & 2.4 and also edited `gemspec` and `.travis.yml` too.